deploy/examples/sqlitevfs-client.yaml

```diff
@@ -0,0 +1,182 @@
+
+# Deployment - sqlitevfs
+#
+# A deployment that uses libcephsqlite (https://docs.ceph.com/en/latest/rados/api/libcephsqlite/)
+# via a pre-created ceph client (via the CephClient CRD).
+#
+# This deployment performs lengthy setup via it's init container to prepare credentials for use
+# credentials are stored on an ephemeral in-memory emptyDir volume between the initContainer and the workload
+#
+# Once the deployment is running, you may `kubectl exec` into the container and use sqlite like so:
+#
+# ```
+# export CEPH_CONF=/libsqliteceph/ceph.conf;
+# export CEPH_KEYRING=/libsqliteceph/ceph.keyring;
+# export CEPH_ARGS=`cat /libsqliteceph/ceph.args`;
+# sqlite3
+# ```
+#
+# Once in the SQLite console:
+# sqlite> .load libcephsqlite.so
+# sqlite> .open file:///yourpoolname:/sqlitevfs.db?vfs=ceph
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sqlitevfs
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sqlitevfs
+      component: app
+  template:
+    metadata:
+      labels:
+        app: sqlitevfs
+        component: app
+    spec:
+      serviceAccountName: sqlitevfs-setup
+
+      initContainers:
+        ## Setup Ceph SQLite VFS
+        - name: setup
+          image: bitnami/kubectl:1.21.11
+          command:
+            - /bin/bash
+            - -c
+            - |
+              # Wait until sqlitevfs cephclient is ready
+              SECRET_NAME=`kubectl get cephclient ${CEPH_CLIENT_NAME} -o jsonpath='{.status.info.secretName}' -n ${CEPH_NS}`
+              while [ -z "${SECRET_NAME}" ]; do
+                sleep 3;
+                echo -e "[info] checking for cephclient [${CEPH_CLIENT_NAME}]...";
+                SECRET_NAME=`kubectl get cephclient ${CEPH_CLIENT_NAME} -o jsonpath='{.status.info.secretName}' -n ${CEPH_NS}`;
+              done;
+
+              echo -e "[info] found ceph client secret [${SECRET_NAME}] in namespace [${CEPH_NS}], extracting content...";
+              SECRET_CONTENT=`kubectl get secret ${SECRET_NAME} -o jsonpath="{.data.${CEPH_CLIENT_NAME}}" -n ${CEPH_NS} | base64 -d`;
+
+              echo -e "[info] writing ceph keyring...";
+              echo -e "[client.${CEPH_CLIENT_NAME}]" > /libsqliteceph/ceph.keyring;
+              echo -e "  key = ${SECRET_CONTENT}" >> /libsqliteceph/ceph.keyring;
+              echo -e "  caps mon = 'allow r'" >> /libsqliteceph/ceph.keyring;
+              echo -e "  caps osd = 'profile rbd pool=raid1, profile rbd pool=rook-raid-1-3n-host'" >> /libsqliteceph/ceph.keyring;
+
+              echo -e "retrieving mon endpoints...";
+              MON_ENDPOINTS=`kubectl get configmap rook-ceph-mon-endpoints -o jsonpath='{.data.data}' -n ${CEPH_NS} | sed 's/.=//g'`;
+
+              echo -e "[info] writing out ceph.conf...";
+              echo -e "[global]" > /libsqliteceph/ceph.conf;
+              echo -e "mon_host=${MON_ENDPOINTS}" >> /libsqliteceph/ceph.conf;
+              echo -e "#debug ms = 1/6" >> /libsqliteceph/ceph.conf;
```

Comment:
How about removing these commented out settings?

Reply:
- [ ] remove commented out lines in ceph.conf